packages/strapi-design-system/src/Field/FieldLabel.js

```diff
@@ -40,9 +39,10 @@ export const FieldLabel = ({ children, required, action, ...props }) => {
 };
 
 FieldLabel.defaultProps = {
-  required: false,
   action: undefined,
+  required: undefined,
 };
+
 FieldLabel.propTypes = {
   action: PropTypes.element,
   children: PropTypes.node.isRequired,
```

Comment:
Can't comment on the line, but i would add jsdoc deprecated comment too :)

Reply:
below this line, above the required prop just to be clear.

Reply:
ahh **regrets** I actually did before removing because I realized the component doesn't have its own storybook and it won't show but maybe I should add it anyway? 👀

Reply:
It should show in IDEs 🙏🏻